lib/active_shipping/carriers/usps.rb

```diff
@@ -654,6 +660,71 @@ def parse_tracking_info(response, node)
       )
     end
 
+    def build_address_standardization_request(addresses)
+      xml_builder = Nokogiri::XML::Builder.new do |xml|
+        xml.AddressValidateRequest('USERID' => @options[:login]) do
+        xml.IncludeOptionalElements(true)
+          xml.ReturnCarrierRoute(true)
+          Array(addresses).each_with_index do |address, id|
```

Comment:
I don't believe you need to wrap `addresses` in an `Array()` constructor; do we anticipate it ever being not-`Array`-like?